package.json

```diff
@@ -14,7 +14,9 @@
     "postbuild-app": "uglifyjs dist/index.js -m -c > dist/index.min.js && rm dist/index.js",
     "watch:app": "watch 'npm run dev-build-app' src --wait=1",
     "watch:css": "watch 'npm run build-css' src/components",
-    "start": "npm run build && parallelshell 'npm run watch:app' 'npm run watch:css' 'npm run watch:spec-runners'",
+    "start": "npm run build && npm run start:server && npm run start:watch",
```

Comment:
I could also remove `npm run start:server` from `start` script. For those of us who have already a web-server running. wdyt?